05/DB.test.js

```diff
@@ -1 +1,86 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+
+describe(".insert()", () => {
+	it("should return 2 when one item is inserted", () => {
+		const db = new DB();
+		const promise = db.insert({ a: 1, b: 2 });
+
+		return promise.then(() => {
+			expect(db._rows.length).toBe(2);
+		});
```

Comment:
Przy takim rozwiązaniu warto dodać `expect.assertions(1)` bo nasz test jest warunkowy (tylko jak będzie prawda). Jeśli będzie błąd to nie mamy porównania i test nie pokaże błędu bo nie wie czego się spodziewać. Natomiast  `expect.assertions(1)` wskazuje, że jedno porównanie się ma wykonać jeśli tak nie będzie to zgładzaj błąd.

PS. Poniżej zrzut z dokumentcji

![image](https://user-images.githubusercontent.com/55095257/229995109-ab2f2840-c112-4b8d-b14c-f8265d025192.png)